dbms/src/Debug/MockRaftStoreProxy.h

```diff
@@ -193,12 +238,19 @@ struct MockRaftStoreProxy : MutexLockWrap
         uint64_t region_id,
         uint64_t to);
 
+    MockRaftStoreProxy()
+    {
+        log = &Poco::Logger::get("MockRaftStoreProxy");
+        table_id = 1;
+    }
 
     std::unordered_set<uint64_t> region_id_to_drop;
     std::unordered_set<uint64_t> region_id_to_error;
     std::map<uint64_t, MockProxyRegionPtr> regions;
     std::list<std::shared_ptr<RawMockReadIndexTask>> tasks;
     AsyncWaker::Notifier notifier;
+    TableID table_id;
+    Poco::Logger * log;
```

Comment:
DB::LoggerPtr is preferred than Poco::Logger
```suggestion
    LoggerPtr log;
```